### PR TITLE
[OSDEV 562] Part 1 remove code related to ppe fields from `/src/app` 

### PR DESCRIPTION
**Pull request by mazursasha1990**
[OSDEV 562](https://opensupplyhub.atlassian.net/browse/OSDEV-562) Remove PPE fields and indexing.
Part 1
- remove code related to ppe fields from `/src/app`

### CODE REVIEW

**Review by killev — DISMISSED** [review]
Changes look relevant to me, but needs to be verified by @vladsha-dev and @VadimKovalenkoSNF

**Review by vladsha-dev — CHANGES_REQUESTED** [review]
I found that there are still functionality or code related to the ppe fields in the code base.
1. For instance, check folder `./src/dedupe-hub/api/app/database/models` for ppe text.
2. You can paste _ppe_ text and search through the all code base. You will find planty of places with the ppe-related code.
Screenshot 2024-02-13 at 09 51 19
3. Since there are ppe-related fields in the database models, please do not forget to prepare the new migrations for the affected models, and fix indexing of the api_facilityindex table.

File: doc/release/RELEASE-NOTES.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * [OSDEV-835](https://opensupplyhub.atlassian.net/browse/OSDEV-835) - Since the FacilityIndex model is primarily used to store cached facility data and display it publicly via the `/facilities/{id}` API endpoint, only public data can be shown. Therefore, caching emails to the FacilityIndex model was removed from the PostgreSQL indexing functions. All instances where emails are publicly displayed have been removed. The only remaining field is `ppe_contact_email`, but all functionality and code related to PPE will be deleted in this [OSDEV-562](https://opensupplyhub.atlassian.net/browse/OSDEV-562) ticket.
 
 ### Code/API changes
-* *Describe code/API changes here.*
+* [OSDEV-562](https://opensupplyhub.atlassian.net/browse/OSDEV-562) - Remove code related to PPE field from app
```
Comment (killev) [inline doc/release/RELEASE-NOTES.md:21]: `form react app` or from `/src/app` (we have inconsistence in naming so it's important to mention what app do you mean)

File: doc/release/RELEASE-NOTES.md [hunk 2]
```diff
@@ -32,6 +32,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * API. Enable token generation based on API permissions in Django. [OSDEV-729](https://opensupplyhub.atlassian.net/browse/OSDEV-729). Updated Settings page to show/hide token tab by user groups. Forbid access to generate token for API if user didn't have permission groups.
 * [OSDEV-219](https://opensupplyhub.atlassian.net/browse/OSDEV-219). Data moderator can merge potential match facilities from Confirm / Reject screen.
 * [OSDEV-835](https://opensupplyhub.atlassian.net/browse/OSDEV-835) - Remove the display of emails in the `activity_reports` section of the `facilities/{id}` API endpoint, as email information is private.
+* [OSDEV-562](https://opensupplyhub.atlassian.net/browse/OSDEV-562) - Remove code related to PPE field from app
```
Comment (killev) [inline doc/release/RELEASE-NOTES.md:35]: The same as above

**Review by killev — CHANGES_REQUESTED** [review]
LGTM, but tiny change requested to be added to release notes

File: doc/release/RELEASE-NOTES.md (hunk 2 above)
Comment (mazursasha1990) [reply doc/release/RELEASE-NOTES.md]: Done

File: doc/release/RELEASE-NOTES.md (hunk 1 above)
Comment (mazursasha1990) [reply doc/release/RELEASE-NOTES.md]: Done

**Review by killev — APPROVED** [review]
LGTM, Approved

**Review by vladsha-dev — APPROVED** [review]
Nice job @mazursasha1990 !
LGTM

**Pull request merged**